README.md

```diff
@@ -222,105 +222,105 @@ environment:
 
 ### Timezone
 
-The default timezone is not defined (aka `UTC`). You can modify the default timezone by the `TZ` environment variable
-with timezone name as value (e.g.: `Europe/Prague`).
+The default timezone is not defined (`UTC` will be used). You can modify the default timezone by setting the `TZ` environment variable
+to the desired timezone name (e.g.: `Europe/Prague`).
 
-You can put it directly with `docker run`:
+It can also be specified directly with `docker run`:
 
 ```shell
 docker run -it --rm -e TZ=Europe/Prague jakubboucek/lamp-devstack-php
 ```
 
-You can also put it to `docker-compose.yml` file:
+Or in your `docker-compose.yml` file:
 
 ```yaml
 environment:
     TZ: Europe/Prague
 ```
 
-The `TZ` environment variable is recognized by Linux, by that variable you modify the default timezone for the whole Linux
+The `TZ` environment variable is recognized by Linux tools as well, by creating the variable you modify the default timezone for the whole Linux
 operating system, PHP, and also MySQL.
 
-### Temporary, Upload and Session storage directory
+### temporary, upload and session storage directory
```

Comment:
Temporary s velkým "T" kvůli konzistenci

Reply:
Všude jinde by mělo být malý "t", ale přepisoval jsem to a možná mi to někde zůstalo - jediný "T" je někde v seznamu na začátku, jako v nadpisu.